locale/en.po

```diff
@@ -5312,20 +5316,40 @@ msgstr "Saving is not currently possible due to:"
 msgid "SAVING_SUCCEEDED"
 msgstr "Saving succeeded"
 
-#: ../src/general/OptionsMenu.tscn:1353 ../src/general/OptionsMenu.tscn:1354
+#: ../src/general/OptionsMenu.tscn:1358 ../src/general/OptionsMenu.tscn:1359
 msgid "SCALING_NONE"
 msgstr "None"
 
-#: ../src/general/OptionsMenu.tscn:1354
+#: ../src/general/OptionsMenu.tscn:1359
 msgid "SCALING_ON"
 msgstr "Scaled"
 
-#: ../src/general/OptionsMenu.tscn:1354
+#: ../src/general/OptionsMenu.tscn:1359
 msgid "SCALING_ON_INVERSE"
 msgstr "Inversely scaled"
 
-#: ../src/general/OptionsMenu.tscn:1485 ../src/general/OptionsMenu.tscn:1518
-#: ../src/general/OptionsMenu.tscn:1519
+#: ../src/general/OptionsMenu.tscn:1939
+msgid "SCREEN_EFFECT"
+msgstr "Screen Effect"
+
+#: ../simulation_parameters/common/screen_effects.json:13
+msgid "SCREEN_EFFECT_GAMEBOY"
+msgstr "Gameboy"
```

Comment:
Just *maybe* this should be abbreviated as "GB" as the trademark on Gameboy might still be valid and messing with the big N for no reason is not the best idea.

Reply:
The trademark is valid and we have 3 options:

1. Ignore it, as here it would probably come to nothing. Also I'm not a lawyer, but this feels [like fair use](https://en.wikipedia.org/wiki/Nominative_use).
2. Write something along the lines of on the bottom of the menu: "Gameboy(R) and Gameboy Color(R) are registered trademarks of Nintendo Of America Inc"
3. Change it to how you suggest

Reply:
In emulation scene most people use abbreviations, which I think is the safest route. So if anything is done, that option should be picked.